save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/vuln/VulnerabilityChangesTab.kt

```diff
@@ -0,0 +1,134 @@
+@file:Suppress("FILE_NAME_MATCH_CLASS", "HEADER_MISSING_IN_NON_SINGLE_CLASS_FILE")
+
+package com.saveourtool.save.frontend.components.views.vuln
+
+import com.saveourtool.save.entities.cosv.CosvFileDto
+import com.saveourtool.save.frontend.externals.diffviewer.reactDiffViewer
+import com.saveourtool.save.frontend.utils.*
+import com.saveourtool.save.utils.toUnixCalendarFormatWithSeconds
+
+import com.saveourtool.osv4k.RawOsvSchema
+import react.FC
+import react.Props
+import react.dom.html.ReactHTML.div
+import react.useState
+import web.cssom.ClassName
+
+import kotlinx.coroutines.await
+import kotlinx.serialization.encodeToString
+import kotlinx.serialization.json.Json
+
+private val json = Json { prettyPrint = true }
+
+val vulnerabilityChangesTab: FC<VulnerabilityChangesTabProps> = FC { props ->
+
+    val (cosvVersions, setCosvVersions) = useState(emptyList<CosvFileDto>())
+
+    val (oldCosvFile, setOldCosvFile) = useState<CosvFileDto?>(null)
+    val (newCosvFile, setNewCosvFile) = useState<CosvFileDto?>(null)
+
+    val (oldVersion, setOldVersion) = useState("")
+    val (newVersion, setNewVersion) = useState("")
+
+    @Suppress("GENERIC_VARIABLE_WRONG_DECLARATION")
+    val newValueRequest = useDeferredRequest {
+        val cosvValueNew: String = get(
+            url = "$apiUrl/cosv/cosv-content?cosvFileId=${newCosvFile?.id}",
+            headers = jsonHeaders,
+            loadingHandler = ::loadingHandler,
+        )
+            .unsafeMap {
+                it.text().await()
+            }
+
+        // FixMe: JSON.stringify() don't work in kotlinJs
+        val decodeValue = Json.decodeFromString<RawOsvSchema>(cosvValueNew)
```

Comment:
@nulls will be like: 

![image](https://github.com/saveourtool/save-cloud/assets/58667063/2508f7b1-1ef0-4106-975f-86c6d34d25ea)